test/test_kaldi_dirs.py

```diff
@@ -0,0 +1,305 @@
+import pytest
+import os.path
+import contextlib
+from pathlib import Path
+
+import lhotse
+import lhotse.utils
+from lhotse.utils import is_module_available
+
+FIXTURE_PATH = Path(os.path.realpath(__file__)).parent / "fixtures"
```

Comment:
```suggestion
FIXTURE_PATH = Path(__file__).resolve().parent / "fixtures"
```

and you can use
```python
Path('xxx').is_file()  # or Path('xxx').exists()
```
in the following and remove `import os.path`.